### ISSUE TITLE
Roles usable in Layman access rights

### ISSUE THREAD

**Issue by index-git** [LayerManager/layman#165]
As a user, I want to be able to use roles in access rights lists, so I do not have to name each user every time for every publication.
We can use roles from GeoServer.

**Comment by jirik** [comment]
Consider what to use for authorization:
- GeoServer
- MapProxy

**Comment by jirik** [comment]
We will use GS, because MapProxy can not be used for WFS

**Comment by jirik** [comment]
We will use [JDBC Role Service](https://docs.geoserver.org/stable/en/user/security/usergrouprole/roleservices.html#jdbc-role-service), on production typically provide by Wagtail.
